intel_extension_for_transformers/neural_chat/pipeline/plugins/audio/tts.py

```diff
@@ -82,7 +80,7 @@ def __init__(self, output_audio_path="./response.wav", voice="default", stream_m
         self.cpu_pool = None
         if not torch.cuda.is_available():
             # ipex IOMP hardware resources
-            if 'LD_PRELOAD' in os.environ and 'libiomp' in os.environ['LD_PRELOAD']:
+            if self.device == 'cpu' and 'LD_PRELOAD' in os.environ and 'libiomp' in os.environ['LD_PRELOAD']:
```

Comment:
@lvliang-intel  @Spycsh  I'd like to suggest not include env variables in the code? Instead, we can make the code simple and add such settings in notebook/readme

Reply:
Will check that soon.

Reply:
@hshen14 this step is a mandatory check for faster TTS on CPU. It is a shared library pre-loaded and I think the only way to check that is to check the LD_PRELOAD. Perf report shows that binding to 24 cores is the best and I do not recommend to remove that because there are nowhere to configure this setting without modifying the code.

Reply:
No, these should not be part of core code.

Reply:
@hshen14 , OK thanks, I will remove this.